packages/cdc-react/src/components/ProgressTracker/ProgressTracker.tsx

```diff
@@ -0,0 +1,62 @@
+import "./ProgressTracker.scss";
+
+export interface ProgressTrackerProps {
+  isComplete?: boolean;
+  currentAmount?: number;
+  isIndeterminate?: boolean;
+  label?: string;
+  totalAmount?: number;
+}
+
+export const ProgressTracker = ({
+  isComplete = false,
+  currentAmount = 0,
+  isIndeterminate = false,
+  label = "",
+  totalAmount = 0,
+}: ProgressTrackerProps) => {
+  const getBarClass = () => {
+    const classList = ["progress-bar-fill"];
+    if (isComplete) classList.push("complete");
+    if (isIndeterminate) classList.push("indeterminate");
+
+    return classList.join(" ");
+  };
+
+  const getBarPercentage = (): number => {
+    if (isIndeterminate || isComplete) return 100;
+
+    currentAmount = Math.max(0, Math.min(currentAmount, totalAmount));
+    totalAmount = Math.max(0, totalAmount);
+
+    const percentage =
+      totalAmount > 0 ? (currentAmount / totalAmount) * 100 : 0;
+
+    return Math.floor(percentage);
```

Comment:
I think making this `Math.floor(percentage).toFixed(2);` would be best for display purposes.